### PR TITLE
fix: added className Prop.

### PR DESCRIPTION
**Pull request by rakesh-m-r**
Bug fix

Currently there is no easy way to customize the badge component.
Also needed for styling the inner contents of the badge, otherwise would have to wrap the child in additional tags.

#339 

Added overlayClasses prop which can accept class names as string and apply to Badge

```
<Badge overlayClasses=" border-teal-50 border-2">
  Hello world
</Badge>
```

Styling of inner content can be done by passing span element with custom styles.

### CODE REVIEW

File: src/components/Badge/Badge.tsx [hunk 1]
```diff
@@ -47,6 +49,9 @@ function Badge({ color = 'brand', children, size, dot }: Props) {
   if (size === 'large') {
     classes.push(__styles.size.large)
   }
+  if(overlayClasses){
```
Comment (MildTomato) [inline src/components/Badge/Badge.tsx:52]: change prop name to be `className` ?

other than that, should be fine.

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: src/components/Badge/Badge.tsx (hunk 1 above)
Comment (rakesh-m-r) [reply src/components/Badge/Badge.tsx]: above changes are pushed.

File: src/components/Badge/Badge.stories.tsx [hunk 2]
```diff
@@ -88,3 +90,7 @@ withDotLarge.args = {
   size: 'large',
   dot: true,
 }
+
+withCustomClassNames.args = {
+  className: ' border-teal-50 border-2'
```
Comment (MildTomato) [inline src/components/Badge/Badge.stories.tsx]: `border-teal-50` this shouldn't be valid.

would have to be `border-teal-100`

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: src/components/Badge/Badge.tsx (hunk 1 above)
Comment (MildTomato) [reply src/components/Badge/Badge.tsx]: nice one!

**Comment by MildTomato** [comment]
sorry, last change request.
then can merge :D

**Comment by rakesh-m-r** [comment]
requested chnage is pushed.

**Review by MildTomato — APPROVED** [review]

**Comment by MildTomato** [comment]
excellent @rakesh-m-r thank you very much 🚀 🔥

**Pull request merged**